redash/settings/organization.py

```diff
@@ -27,6 +27,7 @@
 TIME_FORMAT = os.environ.get("REDASH_TIME_FORMAT", "HH:mm")
 INTEGER_FORMAT = os.environ.get("REDASH_INTEGER_FORMAT", "0,0")
 FLOAT_FORMAT = os.environ.get("REDASH_FLOAT_FORMAT", "0,0.00")
+NULL_VALUE = os.environ.get("REDASH_NULL_VALUE", "null")
```

Comment:
Let's make the default an empty string so that only changes if this parameter is set in `.env`

Reply:
@eradman 

Thank you for your comment!

I think showing NULL by default is preferable for most users because NULL value is visible in most DWH/database visibility tools and web services(both commercial and open source) including:
 - BigQuery
 - Snowflake
 - Databricks
 - Looker Studio
 - phpMyAdmin
 - Sequel Ace
 - Athena
 - superset
 - metabase, etc.

Even on Databricks SQL (that is based on Redash I think), NULL value is visible.

May I hear your thought and/or preference, if possible ?

Thank you !

Reply:
I see your point, and I would also find it useful to be able to see NULL. If we get pushback from users we can reconsider this default.

Reply:
@eradman 
Thank you for seeing !